aeon/dj_pipeline/streams.py

```diff
@@ -22,13 +20,21 @@
 schema = dj.schema(schema_name)
 
 
-class DeviceConfig(NamedTuple):
+class DeviceConfig:
```

Comment:
I think we can remove the use of this `DeviceConfig` class completely, as it seems to only serve the purpose of being a namespace

Reply:
hmm.. ok let me try & re-structure